commands/remove/yarn.lock

```diff
@@ -0,0 +1,2862 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Definitely remove this, please. Lerna itself uses npm, and in any case shouldn't be developing from a leaf dir.